source/install/install-kubernetes.rst

```diff
@@ -83,13 +136,43 @@ Deploy Mattermost
-    
-  Additional fields are documented `in the example <https://github.com/mattermost/mattermost-operator/blob/master/docs/examples/mattermost_full.yaml>`__. If you have previous experience with Kubernetes Custom Resources, you can also check the `Custom Resource Definition <https://github.com/mattermost/mattermost-operator/blob/master/config/crd/bases/installation.mattermost.com_mattermosts.yaml>`__.
+    "spec.image", "Docker image for the Mattermost app servers."
+    "spec.imagePullPolicy", "Image Pull policy used by Mattermost deployment."
+    "spec.useServiceLoadBalancer", "Set to true to use AWS or Azure load balancers instead of an NGINX controller."
+    "spec.serviceAnnotations", "Service annotations to use with AWS or Azure load balancers."
+    "spec.ingress.enabled", "Indicates if Ingress should be created by Mattermost Operator."
+    "spec.ingress.annotations", "Custom annotations propagated to Ingress resource."
+    "spec.ingress.tlsSecret", "Name of a Secret that contains TLS certificates for the ingress."
+    "spec.database.external.secret", "Name of a Kubernetes secret containing connection string to an external database."
+    "spec.fileStore.external.url", "External File Storage URL."
+    "spec.fileStore.external.bucket", "File Storage bucket name."
+    "spec.fileStore.external.secret", "Name of a Kubernetes secret that contains credentials to external file storage."
+    "spec.elasticSearch.host", "Elasticsearch hostname."
+    "spec.elasticSearch.username", "Username to log into Elasticsearch."
+    "spec.elasticSearch.password", "Password to log into Elasticsearch."
+    "spec.scheduling.resources", "Resource requests and limits of pod and container (Kubernetes)."
+    "spec.scheduling.nodeSelector", "Node selector for assigning pods to nodes (Kubernetes)."
+    "spec.scheduling.affinity", "Affinity and anti-affinity for assigning pods to nodes (Kubernetes)."
+
+  More information on these additional fields can be found `in this example <https://github.com/mattermost/mattermost-operator/blob/master/docs/examples/mattermost_full.yaml>`__. If you have previous experience with Kubernetes Custom Resources, you can also check the `Custom Resource Definition <https://github.com/mattermost/mattermost-operator/blob/master/config/crd/bases/installation.mattermost.com_mattermosts.yaml>`__.
+
+3. Create a file ``mattermost-database-secret.yaml`` to house secrets related to your installation's database. The database secret needs to be created in the namespace that will hold the Mattermost installation. The secret should contain the following data:
+
+  .. code-block:: yaml
+
+    apiVersion: v1
+    data:
+      DB_CONNECTION_CHECK_URL: <DB_CONNECTION_CHECK_URL>
+      DB_CONNECTION_STRING: <DB_CONNECTION_STRING>
+      MM_SQLSETTINGS_DATASOURCEREPLICAS: <MM_SQLSETTINGS_DATASOURCEREPLICAS>
+    kind: Secret
+    metadata:
+      name: my-postgres-connection
```

Comment:
nit: you noted that the metadata.name value should be set below in the filestore secret. We should probably do the same here.

Reply:
@nickmisasi - Has this feedback been incorporated into the PR?

Reply:
Latest commit has the update.